.github/ISSUE_TEMPLATE/technical_improvement.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Tech improvement
+about: Create a report to help us improve
+title: "[TECH]"
```

Comment:
Idem as above..

Reply:
I don't think it would happen because github set the focus to it when a new issue with template TECH is created